courses/free-courses-en.md

```diff
@@ -844,6 +844,8 @@
 * [ReactJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - Codevolution
 * [Start Using React to Build Web Applications](https://egghead.io/courses/react-fundamentals) - Joe Maddalone
 * [The Beginner's Guide to React](https://egghead.io/courses/the-beginner-s-guide-to-react) - Kent C. Dodds
+* [React](https://syllabus.migracode.org/courses/introduction-3/course-content/react) - MigraCode Barcelona
```

Comment:
Looks like this is just a summary. Would it be a value add?

Reply:
It's interesting that this looks like a summary to you. It's the index page of the React module where you could click and enter the 3+1 React syllabus. ( 3 courses + 1 extra material)

Reply:
It's really more like the thrid lesson of the "Web Development Course". will move it out of React.